replication/li_wand_2016/.gitignore

```
@@ -0,0 +1 @@
+images/
```

Add a newline at the end of the file.

```suggestion
images/

```